### DISCUSSION TITLE
Connexion directe à l'interface d'administration depuis une interface WAN

### DISCUSSION THREAD

**Discussion by romainbacque** [Ysurac/openmptcprouter#2319] [Q&A]
Bonjour,

J'utilise OpenMPTCProuter pour agréger plusieurs connexions à internet été j'ai effectué très peu de modifications sur les paramètres par défaut pour le faire fonctionner (à merveille, d'ailleurs, bravo au développeur !).

L'une de mes interfaces dispose d'une adresse IP publique, c'est à dire disponible directement depuis internet, comme on peut le voir dans la capture ci-dessous. Cette IP publique a vocation, entre autres, à me permettre d'accéder directement à l'interface d'administration du routeur sans passer par la VPS ni un VPN.

<img width="470" alt="Capture d’écran, le 2022-04-11 à 12 13 47" src="https://user-images.githubusercontent.com/13886382/162719882-8d919c1c-8762-47a3-95e6-35d55f400540.png">

J'ai essayé de configurer le pare-feu, en vain, pour rediriger les ports 80 et 443 (source : WAN) vers l'adresse IP du routeur (l'adresse sur le réseau interne, par défaut 192.168.100.1). Impossible de me connecter à la page d'administration (pas de réponse du serveur). D'ailleurs, lorsque je crée une redirection sur les mêmes ports avec pour source VPN et que j'essaie la même manip en me connectant à l'IP de la VPS, c'est le même problème.

Je dois oublier quelque chose mais impossible de trouver quoi. Quelqu'un aurait-il une piste ?

Merci

**Comment by SebaGaillard74** [comment]
C'est une question que je me suis posée aussi.

En ce qui concerne l'accès via le VPS, oui c'est faisable. Il faut aller dans réseau> Pare-feu> Redirection de ports, et de là créer ta règle en mettant bien en zone source le VPN, et en destination ton IP LAN

En ce qui concerne l'administration via le WANxxx, je ne pense pas que ce soit possible. Si tu regardes ta table de routage, la route par défaut avec la priorité 0 (donc la priorité max) passe par le VPN. Donc quand tu vas interroger via la WAN1, et sauf erreur de ma part, ton routeur va tenter de répondre par le VPN, et ton pare-feu va bloquer la connexion.

` route
Kernel IP routing table
Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
default         10.255.255.1    0.0.0.0         UG    0      0        0 tun0
default         192.168.xx.yy    0.0.0.0         UG    3      0        0 wan1
default         192.168.aa.bb    0.0.0.0         UG    4      0        0 wan2
default         10.255.255.1    0.0.0.0         UG    1200   0        0 tun0`

Si je me mets sur le réseau wan1 avec une machine et que je tente un telnet, après avoir mis en place une redirection de port 443>443, ça marche:

` nc -z 192.168.xx.yy 443
Connection to 192.168.xx.yy 443 port [tcp/https] succeeded!`

en revanche depuis internet... ça ne marche pas

`nc -z xxxx.xxxx.xxxx.xxxx 443
<pas de réponse>` 

Est-ce que tu peux me montrer un screen de ta redirection de port?
